nupic/research/frameworks/vernon/handlers.py

```diff
@@ -1059,14 +1059,352 @@ def get_execution_order(cls):
         return eo
 
 
+class MetaContinualLearningExperiment(SupervisedExperiment):
+    """
+    Experiment class for meta-continual learning (based on OML and ANML meta-continual
+    learning setups). There are 2 main phases in a meta-continual learning setup.
+
+        - meta-training: Meta-learning representations for continual learning over all
+        tasks
+        - meta-testing: Montinual learning of new tasks and model evaluation
+
+    More specifically, each phase is divided into its own training and testing phase,
+    hence we have 4 such phases.
+
+        - meta-training training: Train the inner loop learner (i.e., slow parameters)
+        for a specific task
+        - meta-training testing: Train the outer loop learner (i.e., fast parameters)
+        by minimizing the test loss
+
+        - meta-testing training: Train the inner loop learner continually on a sequence
+        of holdout tasks
+        - meta-testing testing: Evaluate the inner loop learner on the same tasks
+
+    The parameters for a model used in a meta-continual learning setup are broken down
+    into 2 groups:
+
+        - fast parameters: Used to update slow parameters via online learning during
+        meta-training training, and updated along with slow parameters during
+        meta-training testing
+        - slow parameters: Updated during the outer loop (meta-training testing)
+    """
+
+    def setup_experiment(self, config):
+        """
+        Configure the experiment for training
+
+        :param config: Dictionary containing the configuration parameters, most of
+        which are defined in SupervisedExperiment, but some of which are specific to
+        MetaContinualLearningExperiment
+
+            - experiment_class: Class used to run experiments, specify
+            `MetaContinualLearningExperiment` for meta-continual learning
+            - adaptation_lr: Learning rate used to update the fast parameters during
+            the inner loop of the meta-training phase
+            - tasks_per_epoch: Number of different classes used for training during the
+            execution of the inner loop
```

Comment:
Doesn't `tasks_per_epoch` reflect the number of tasks used in the outer loop (absent the replay set)? It may also help to reiterate "one task corresponds to learning one class" or something like that.

Reply:
Yes, that's true, but also how many different tasks/classes the inner loop learner sees before the outer loop update. I think the latter is more informative. And yes, I can add "each task corresponds to learning a different classification label".